.github/workflows/post-merge-dynamo-to-dev.yml

```diff
@@ -36,9 +36,7 @@ jobs:
           aws-region: ${{ env.AWS_REGION }}
 
       - name: SAM Validate
-        run: sam validate --region ${{ env.AWS_REGION }} -t 
-          working-directory: infra-l2-dynamo/**
-          template-file: template.yaml
+        run: sam validate --region ${{ env.AWS_REGION }} -t infra-l2-kms/template.yaml
```

Comment:
this is pointing at kms, should be dynamo?
also working dir should take care of the path so you shouldnt need to set it here again.

although line 24 should be: (remove the /**)
`working-directory: infra-l2-dynamo`